### DISCUSSION TITLE
link to twitter lead to nowhere

### DISCUSSION THREAD

**Discussion by ninyawee** [creatorsgarten/discussions#12] [General]
The twitter link on the github org lead to nowhere.

![image](https://user-images.githubusercontent.com/8089231/221759573-7fa15c40-5e7c-4b05-a560-34b75802dc54.png)

**Comment by dtinth** [comment]
Where did you find the Twitter link? I couldn’t find it…

**Reply by ninyawee** [reply]
https://github.com/creatorsgarten

**Reply by ninyawee** [reply]
![image](https://user-images.githubusercontent.com/8089231/221891960-a3c9a356-827e-471d-ba50-1c68566119ed.png)
here  krub

**Reply by dtinth** [reply]
Thank you krub. I see. We don’t have a Twitter account yet, so I deleted the link now. Thank you for reporting krub!
